### PR TITLE
Shrinking and labels

### PR DESCRIPTION
**Pull request by coot**
@karknu I added labeling to check how look distributions.  The shrinking is not the best, but it's a good starting point.

### CODE REVIEW

**Comment by karknu** [comment]
@coot Could you update shrinking to work with the map format?

**Comment by coot** [comment]
@karknu I will

**Comment by coot** [comment]
@karknu done

File: ouroboros-network/test/Test/Mux/Control.hs [hunk 1]
```diff
@@ -88,6 +90,18 @@ instance Arbitrary ArbitraryVersionFT where
...
+        1 -> Just (Mx.Version1 (Mx.NetworkMagic (fromIntegral y)))
+        _ -> Nothing
+      toVersion (CBOR.TkListLen n : CBOR.TkInteger x : CBOR.TkInt y : _) | n >= 2 && y >= 0 = case x of
+        0 -> Just (Mx.Version0 (Mx.NetworkMagic (fromIntegral y)))
+        1 -> Just (Mx.Version1 (Mx.NetworkMagic (fromIntegral y)))
+        _ -> Nothing
+      toVersion (CBOR.TkListLen n : CBOR.TkInteger x : CBOR.TkInteger y : _) | n >= 2 && y >= 0 = case x of
+        0 -> Just (Mx.Version0 (Mx.NetworkMagic (fromIntegral y)))
+        1 -> Just (Mx.Version1 (Mx.NetworkMagic (fromIntegral y)))
+        _ -> Nothing
```
Comment (karknu) [inline ouroboros-network/test/Test/Mux/Control.hs]: This looks like an incorrect merge to me.

**Comment by karknu** [comment]
LGTM.

**Comment by coot** [comment]
Do you have an idea if `bors r+` will merge to the right branch?

**Comment by karknu** [comment]
Did it pick up on the `b*rs r+` within your comment?

**Comment by coot** [comment]
No, I merged with github button

**Pull request merged**